docs/tutorials/notebooks/sparse_monge_displacements.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
<p>most particle<strong>s</strong></p><p>"<strong>the same coordinate</strong>" -&gt; since some points have more than one sometimes, "the" is a bit ambiguous</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/374/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>